fizzbuzz.CBL

```diff
@@ -0,0 +1,31 @@
+IDENTIFICATION DIVISION.
```

Comment:
I ran this and noticed the output looks a bit weird on the numbers. Can we make those just 1, 2, etc.? 

```
$main
001
002
Fizz
004
Buzz
Fizz
007
008
```

Reply:
The storage picture doesn't allow for 1 and 11 to coexist in the same data storage, I'll change it to `99` from `999`. I'll have a quick refresh to see if it's actually possible to do (should be, right?)